cli/cmd/validate.go

```diff
@@ -0,0 +1,135 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/cloudquery/cloudquery/cli/internal/auth"
+	"github.com/cloudquery/cloudquery/cli/internal/specs/v0"
+	"github.com/cloudquery/plugin-pb-go/managedplugin"
+	"github.com/cloudquery/plugin-pb-go/pb/plugin/v3"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	validateShort   = "Validate spec(s) files"
+	validateExample = `# Validate spec(s) files
+cloudquery validate ./directory
+# Validate specs from directories and files
+cloudquery validate ./directory ./aws.yml ./pg.yml
+`
+)
+
+func NewCmdValidate() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "validate [files or directories]",
+		Short:   validateShort,
+		Long:    validateShort,
+		Example: validateExample,
+		Args:    cobra.MinimumNArgs(1),
+		RunE:    validate,
+		Hidden:  true,
```

Comment:
Hidden on purpose for now to test this a bit before making it public for everyone